t/TestWasm.pm

```diff
@@ -108,9 +108,13 @@ add_block_preprocessor(sub {
 
     my $main_config = $block->main_config || '';
 
-    $block->set_value("main_config",
-                      "env WASMTIME_BACKTRACE_DETAILS=1;\n"
-                      . $main_config);
+    my @env_vars = (
+        "env WASMTIME_BACKTRACE_DETAILS=1",
+        "env RUST_BACKTRACE=full",
+    );
+    my $env_section = join(";\n", @env_vars) . ";\n\n";
+
+    $block->set_value("main_config", $env_section . $main_config);
```

Comment:
Maybe https://metacpan.org/pod/Test::Nginx::Socket#env_to_nginx would be better suited for this task, even though for now I'm more focused on the other points made in the PR thread.